app/views/shared/_collection_exhibit_show.html.erb

```diff
@@ -33,6 +33,6 @@
           <%= current_tab.html_safe if current_tab %>
         </div>
       <% end %>
-    <% end %>
+    <% end if params[:tab] %>
```

Comment:
@mccalluc you added a conditional for `if params[:tab]`. Does this mean that there are cases where `params[:tab]` is nil? If so, then it looks like we'd end up with an empty `<div class="row">`. Should the row be moved into the conditional too?